docs/design/software-architecture.adoc

```diff
@@ -91,6 +91,12 @@ image::http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubuse
 | OpenShift ImageStream
 |
 
+| Task `ods-package-image`
+| Generate image SBOM
+| HTTP / JSON API
+| Trivy
```

Comment:
Is trivy actually downloading anything when generating the SBOM? I thought this could/should be an offline operation. In that case there wouldn't be an interface here.

Reply:
it is interacting with the aqua DBs AFAIS, will double check and come back